server/interview/interview.controller.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+
+const interviewService = require('./interview.service');
+
+//Create interview
+router.post('/', async function(req, res) {
+    const postBody = req.body;
+    const userId = postBody.userId;
```

Comment:
You should be adding an authentication middleware to this route, and be getting the userId from req.user.id - @abohakobyan can assist with this as he made this middleware

Reply:
Added the authentication middleware to the routes